src/confread.c

```diff
@@ -45,22 +45,23 @@
 
 #include "igmpproxy.h"
 
-#define READ_BUFFER_SIZE    512     // Inputbuffer size...
+#define READ_BUFFER_SIZE 512 // Inputbuffer size...
 
 #ifndef MAX_TOKEN_LENGTH
-    #define MAX_TOKEN_LENGTH  30    // Default max token length
+#define MAX_TOKEN_LENGTH 30 // Default max token length
 #endif
 
-FILE            *confFilePtr;       // File handle pointer
-char            *iBuffer;           // Inputbuffer for reading...
-unsigned int    bufPtr;             // Buffer position pointer.
-unsigned int    readSize;           // Number of bytes in buffer after last read...
-char    cToken[MAX_TOKEN_LENGTH];   // Token buffer...
-short   validToken;
+FILE *confFilePtr;             // File handle pointer
+char *iBuffer;                 // Inputbuffer for reading...
+unsigned int bufPtr;           // Buffer position pointer.
+unsigned int readSize;         // Number of bytes in buffer after last read...
+char cToken[MAX_TOKEN_LENGTH]; // Token buffer...
+short validToken;
 
-/**
-*   Opens config file specified by filename.
-*/
+
+/*******************************************************************************
+ * Opens config file specified by filename.
+ ******************************************************************************/
```

Comment:
This reformat (and on other places too) introduce inconsistency. As in other files is used just:
```
/**
*   ...
*   ...
*/
```